pkg/provider/provider_test.go

```diff
@@ -5,38 +5,43 @@ import (
 	"io/ioutil"
 	"testing"
 
-	mk8s "github.com/containous/maesh/pkg/k8s"
 	"github.com/containous/maesh/pkg/topology"
 	"github.com/containous/traefik/v2/pkg/config/dynamic"
 	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-type stateTableMock func(svcPort mk8s.ServicePort) (int32, bool)
+type stateTableMock func(ns, name string, port int32) (int32, bool)
```

Comment:
Can you rename all `ns` param in this file?

```suggestion
type stateTableMock func(namespace, name string, port int32) (int32, bool)
```